### ISSUE TITLE
Wrong description for 1.3.6.1.4.1.311.20.2.3

### ISSUE THREAD

**Issue by michael-o** [lapo-luchini/asn1js#49]
Here: https://github.com/lapo-luchini/asn1js/blob/383db01fbabccfd73b8cdb425efb6510b9131959/oids.js#L1202

Same issue as https://github.com/openssl/openssl/pull/10029 and https://github.com/kaikramer/keystore-explorer/issues/246

=> `userPrincipalName`.

**Comment by lapo-luchini** [comment]
Hi, the source of that file is not managed by me (and I use it in pristine form), so I'd suggest you to write to original author of the data, which you can find in both credited in the README and as a leading comment in oid.js itself; it will fix in this project too, as soon as the new updates got pulled.

**Comment by michael-o** [comment]
Done, waiting for response...

**Comment by michael-o** [comment]
Received confirmation from Peter that he updated his main copy. It will appear with the next update. So we will need to wait for it.

**Comment by lapo-luchini** [comment]
Yep, right now [update.sh](https://github.com/lapo-luchini/asn1js/blob/trunk/update.sh) (which downloads and translates the content) gives:
```
HTTP request sent, awaiting response... 304 Not Modified
File ‘dumpasn1.cfg’ not modified on server. Omitting download.
```
I'll check from time to time but please add a comment here if you notice a change before I do.

**Comment by lapo-luchini** [comment]
(still nothing)

**Comment by michael-o** [comment]
Still wrong :-(

**Comment by lapo-luchini** [comment]
Upstream [`dumpasn1.cfg`](https://www.cs.auckland.ac.nz/~pgut001/dumpasn1.cfg) was updated.
Closed by new import in 75da453c97855f5b424faec7d7243b425cec696f.
